public/modules/analysis/fdf/fdf.rb

```diff
@@ -180,7 +182,7 @@ def check_files_similarity(files, extension)
       # Function used to initialize and run the fdf
       # Results aren't saved because it's done into the framework
       #
-      # @param [OBject] results object
+      # @param result [Object] results object
```

Comment:
`[AnalyseResult]` resulting object of the analysis that will be saved later on, on which to aggregate data